x-pack/legacy/plugins/siem/public/components/navigation/tab_navigation/index.tsx

```diff
@@ -102,7 +112,7 @@ export class TabNavigation extends React.PureComponent<TabNavigationProps, TabNa
         disabled={tab.disabled}
         isSelected={this.state.selectedTabId === tab.id}
         key={`navigation-${tab.id}`}
-        onClick={() => this.handleTabClick(tab.href, tab.id)}
+        onClick={(e: React.MouseEvent) => this.handleTabClick(e, tab.href, tab.id)}
       >
         {tab.name}
```

Comment:
I feel like we should just use a `EuiLink` component inside of the tab like below and we will get all the functionality that we wanted (we just need to override some css to make it look good) 
```
<EuiLink href={tab.href}>{tab.name}</EuiLink>
```
an delete all the code
![image](https://user-images.githubusercontent.com/189600/60212208-a6734300-982e-11e9-962b-6c5a8f642f06.png)
and I think that we will not need the changes in the `link_to` folder.

what do you think?

Reply:
https://github.com/elastic/eui/issues/2077
Maybe we should see what feedback we get here. @chandlerprall says "The two people with Strong Opinions are out this week, but I'm sure you'll get feedback on Monday"

Reply:
Passing any kind of component as the Tab's name isn't supported. It will work in the current form - with a React proptype warning - but it's possible EUI will break that in the future without notice. I believe we have talked about this use case before but I cannot find the conversation.